src/applications/disability-benefits/all-claims/components/PrivateProviderTreatmentView.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { formatReviewDate } from 'us-forms-system/lib/js/helpers';
+
+export default function PrivateProviderTreatmentView({ formData }) {
+  let from = '';
+  let to = '';
+
+  if (formData.treatmentDateRange) {
```

Comment:
This does a check for `treatmentDateRange` here (suggesting it could be missing) but the component will render ` - ` if this doesn't exist